cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/NetworkClientImpl.kt

```diff
@@ -30,13 +33,15 @@ import okhttp3.Request
 import okhttp3.RequestBody
 
 internal fun createNetworkClient(
+    context: Context,
```

Comment:
This logic should not be in the NetworkClient, it should be in the Service class (ServiceImpl).